jekyll/_cci2/orbs-faq.md

```diff
@@ -121,10 +120,14 @@ Question: How can I intentionally fail a job that invokes an orb from within an
 
 Answer: You can always return a non-zero status code from the shell to fail the job. You can also use `run: circleci-agent step halt` as a step to exit the job without failing.
 
+### Private Installation of CircleCI When Using Orbs
+
+* Question: Can I use a private installation of CircleCI when using creating/using orbs?
+
+* Answer: No. Currently the CircleCI server does not support private installations of CircleCI when creating or using orbs.
```

Comment:
1. Do we capitalize Server, the product?
2. "private installations of orbs." (I don't think we need the whole end of this sentence, it's a bit confusing.